src/components/AssetSearch/helpers/filterAssetsBySearchTerm/filterAssetsBySearchTerm.test.ts

```diff
@@ -1,38 +1,39 @@
+import { Asset } from '@shapeshiftoss/asset-service'
 import { filterAssetsBySearchTerm } from './filterAssetsBySearchTerm'
 
 const assets = [
   {
-    address: '0x9124248f2AD8c94fC4a403588BE7a77984B34bb8',
+    tokenId: '0x9124248f2AD8c94fC4a403588BE7a77984B34bb8',
     symbol: 'ETH',
     name: 'Ethereum'
   },
-  { address: '3Er42Ft2ZAyAbrpQ1PJUfgF7LbdJztcycd', symbol: 'BTC', name: 'Bitcoin' },
+  { tokenId: '3Er42Ft2ZAyAbrpQ1PJUfgF7LbdJztcycd', symbol: 'BTC', name: 'Bitcoin' },
   {
-    address: '0x79be75ffc64dd58e66787e4eae470c8a1fd08ba4',
+    tokenId: '0x79be75ffc64dd58e66787e4eae470c8a1fd08ba4',
     symbol: 'AAMMDAI',
     name: 'Aave AMM DAI'
   },
-  { address: '0x6b175474e89094c44da98b954eedeac495271d0f', symbol: 'DAI', name: 'DAI' }
+  { tokenId: '0x6b175474e89094c44da98b954eedeac495271d0f', symbol: 'DAI', name: 'DAI' }
 ]
 
 describe('filterAssetsBySearchTerm', () => {
   it('returns based on symbol', () => {
-    const returnedAssets = filterAssetsBySearchTerm('btc', assets)
+    const returnedAssets = filterAssetsBySearchTerm('btc', assets as unknown as Asset[])
     expect(returnedAssets[0].symbol).toBe('BTC')
   })
 
   it('returns based on displayName', () => {
-    const returnedAssets = filterAssetsBySearchTerm('Bitcoin', assets)
+    const returnedAssets = filterAssetsBySearchTerm('Bitcoin', assets as unknown as Asset[])
     expect(returnedAssets[0].name).toBe('Bitcoin')
   })
 
   it('returns based on address', () => {
-    const returnedAssets = filterAssetsBySearchTerm(assets[0].address, assets)
+    const returnedAssets = filterAssetsBySearchTerm(assets[0].tokenId, assets as unknown as Asset[])
     expect(returnedAssets[0].symbol).toBe('ETH')
   })
 
   it('returns closest match instead of the first one in the array', () => {
-    const returnedAssets = filterAssetsBySearchTerm('DAI', assets)
+    const returnedAssets = filterAssetsBySearchTerm('DAI', assets as unknown as Asset[])
```

Comment:
I would be interested to know what is returned when there are multiple matches like Harmony (ONE) and Menlo (ONE)

Reply:
I could write a test for that. I think the reason it was not written is because we use a package https://github.com/kentcdodds/match-sorter that handles that logic and it favors the name over the symbol right now.

Reply:
just curious why we're casting here?

Reply:
Its because in the test the assets that are defined only have the fields that it needs and not the whole asset object so I did that to get around the linter in the test.

Reply:
@0xdef1cafe I was able to just pick the fields off the type I needed so that defining the object in the test allowed me just to use what I needed.